test/integration/targets/connection_ssh/runme.sh

```diff
@@ -79,3 +79,9 @@ ANSIBLE_CONFIG=./test_ssh_defaults.cfg ansible-playbook verify_config.yml "$@"
 # ensure we handle cp with spaces correctly, otherwise would fail with
 # `"Failed to connect to the host via ssh: command-line line 0: keyword controlpath extra arguments at end of line"`
 ANSIBLE_SSH_CONTROL_PATH='/tmp/ssh cp with spaces' ansible -m ping all -e ansible_connection=ssh -i test_connection.inventory "$@"
+
+# Check if control path is different
+ansible -m ping -i control_path.inventory web -vvv 1> stdout.txt 2> /dev/null
+first=$(grep stdout.txt -c "ControlPath=" | head -1)
+second=$(grep stdout.txt -c "ControlPath=" | tail -1)
```

Comment:
I don't think you can swap the args to `grep` like that:
```suggestion
first=$(grep -c "ControlPath=" stdout.txt | head -1)
second=$(grep -c "ControlPath=" stdout.txt | tail -1)
```
```console
10:20 ++ grep stdout.txt -c ControlPath=
10:20 ++ head -1
10:20 grep: ControlPath=: No such file or directory
10:20 + first=
10:20 ++ tail -1++ grep stdout.txt -c ControlPath=
10:20 
10:20 grep: ControlPath=: No such file or directory
10:20 + second=
10:20 + test '' '!=' ''
```
*(https://dev.azure.com/ansible/ansible/_build/results?buildId=126113&view=logs&j=9c02643b-033d-5485-b509-6aae1ba83dd6&t=c1f7e8ad-ec35-5c07-81d3-29104585d5fa&l=14471)*